preview-comment/action.yml

```diff
@@ -25,6 +25,10 @@ inputs:
   message-id:
     description: A unique identifier to prevent multiple comments on the same pull request
     required: false
+  github-token:
+    description: GitHub access token to comment on PRs
+    required: false
+    default: ${{ github.token }}
```

Comment:
This is where the magic happens, it uses the `github` context to auto-fill the token on `github-token`.